extensions/cornerstone/src/components/WindowLevelActionMenu/getWindowLevelActionMenu.tsx

```diff
@@ -24,6 +30,12 @@ export function getWindowLevelActionMenu({
       return { [displaySet.Modality]: presets[displaySet.Modality] };
     });
 
+  const modalities = displaySets.map(displaySet => displaySet.Modality);
```

Comment:
This should reduce the set of modalities by excluding the non-WL modalities, and then if the modalities set still contains any element, then window level is permitted.  That is,
SEG => not permitted
SEG, SM => not permitted
SEG, CR => permitted
SEG, CR, OT => permitted
You can do this with an every statement rather than a some.